lib/generators/hyrax/riiif_generator.rb

```diff
@@ -33,7 +33,7 @@ def mount_route
 
   def override_image_url_builder_in_hyrax_config
     insert_into_file 'config/initializers/hyrax.rb', before: /^  # config.iiif_image_url_builder/ do
-      "  config.iiif_image_url_builder = lambda do |file_id, base_url, size|\n" \
+      "  config.iiif_image_url_builder = lambda do |file_id, base_url, size, format|\n" \
```

Comment:
Does this change in the lambda signature require updating documentation or at least need to be called out in the 3.0 migration docs?

Reply:
Yes, I believe it will.